src/main.ts

```diff
@@ -105,17 +110,24 @@ async function addLicenseUrlMappings(args: string[], projectDir: string) {
 async function addLicensePackageMappings(args: string[], projectDir: string) {
   const licenseInfoFile = path.join(projectDir, "licenseInfos.json");
   if (fs.existsSync(licenseInfoFile)) {
-    args.push("--manual-package-information", licenseInfoFile);
+    args.push("--override-package-information", licenseInfoFile);
+  } 
+}
+
+async function addIgnorePackages(args: string[], projectDir: string) {
+  const ignorePackagesFile = path.join(projectDir, "ignorePackages.json");
+  if (fs.existsSync(ignorePackagesFile)) {
+    args.push("--ignored-packages", ignorePackagesFile);
   }
 }
 
 async function addExportOptions(args: string[], exportDir: string, tempExportDir: string) {
```

Comment:
@datlam162 you could maybe change the tempExportDir to be generated in this method to be more consistent to the other methods. in this case you would need to make the tempExportDir as return value because you need it in buildReport later though ;)